lib/whitelistedUrlPrefixes.js

```diff
@@ -27,5 +27,7 @@ module.exports = [
   'https://ads-serve.bravesoftware.com/', // remove this once ads catalog moves to using prod
   'https://pdfjs.robwu.nl/logpdfjs', // allowed because it gets canceled in tracking protection
   'https://publishers-distro.basicattentiontoken.org/',
-  'https://publishers-staging-distro.basicattentiontoken.org/'
+  'https://publishers-staging-distro.basicattentiontoken.org/',
+  'https://extensionupdater.brave.com/',
+  'https://clients2.google.com/' // allowed because it 307's to https://extensionupdater.brave.com
```

Comment:
can we be more specific with this prefix based on https://github.com/brave/brave-core/pull/2493#issuecomment-495721065?

for instance `https://clients2.google.com/service/update2/crx`